.github/workflows/lint.yml

```diff
@@ -15,19 +15,17 @@ jobs:
       - name: Check out Git repository
         uses: actions/checkout@v2
 
-      - name: Set up node
-        uses: actions/setup-node@v1
-        with:
-          node-version: 16.x
-          registry-url: https://registry.npmjs.org
+      - name: Set the correct Node version using nvm
+        shell: bash -l {0} 
+        run: nvm install
 
       - name: Install dependencies
         run: npm ci
 
       - name: Run linters
-        uses: wearerequired/lint-action@v1
+        uses: wearerequired/lint-action@v2
```

Comment:
if you're changing this, it's safer to use a git commit than a major version, as the minor version can be bumped to change the package maliciously.

Reply:
it won't affect package.json or package-lock.json, but pacakge.json/package-lock.json give you stronger guarantees than the version here. you could lock it to a more explicit version by specifying a git commit hash from the wearerequired/lint-action repo.

you don't need to change this now, though.